lua/nvim-lsp-installer/server.lua

```diff
@@ -75,7 +75,9 @@ end
 ---@param opts table @The lspconfig server configuration.
 function M.Server:setup(opts)
     self:setup_lsp(opts)
-    self:attach_buffers()
+    if not (opts.autostart == false) then
+        self:attach_buffers()
+    end
```

Comment:
@mjlbach :wave: what do you think about doing this in lspconfig by default (attaching all open buffers when calling `.setup()`)? Happy to open PR